src/aura/AccountWizardCmp/AccountWizardCmpController.js

```diff
@@ -0,0 +1,40 @@
+({
+    doInit: function(component) {
+        component.set("v.accountId",null);
+    },
+    handleClick: function(component, event){
+        var button = component.get("v.truthy");
+        component.set("v.truthy",true);
+    },
+    handleSubmit: function(component, event){
```

Comment:
Do sprawdzenia czy to jest nam potrzebne - moim zdaniem nie(podobnie jak z tym co robiliśmy na invoicecmp)